### DISCUSSION TITLE
Workflow to divide document into multiple sections.

### DISCUSSION THREAD

**Discussion by abalter** [quarto-dev/quarto-cli#9770] [Q&A]
### Description

I'm looking for a clear description of the correct workflow to divide a single document into multiple files. For example:

_imaginary project folder listing_
```
project/
|_main.qmd
|_introduction.qmd
|_chapter_1.qmd
|_chapter_2.qmd
|_conslusion.qmd
|_images/
```

where _main.qmd_ has code that brings in the rest of the document.

I looked throught the documentation. I specifically read the sections on [Includes](https://quarto.org/docs/authoring/includes.html) and [Managing Execution](https://quarto.org/docs/projects/code-execution.html). I also tried some regular old googling. I haven't found a workflow or recipe that I feel satisfied with.

**Comment by mcanouil** [comment]
You have the book project type or using a single document with include shortcodes.

I don't know what you want but from your "project" you are clearly describing the use of the include shortcode.

**Reply by abalter** [reply]
Like this? `{{< include _content.qmd >}}` or like this `@include "sections/section1.qmd"`?

Although, the book project type looks promising as well. By all appearances, it's automagic. Is that correct?

**Reply by mcanouil** [reply]
> Like this? {{< include _content.qmd >}} or like this @include "sections/section1.qmd"?

What? You said you read the documentation about the include shortcode, did you? Where did you see that weird `@include` syntax exactly?

> Although, the book project type looks promising as well. By all appearances, it's automagic. Is that correct?

See <https://quarto.org/docs/books/> for explanation/description.
